src/terminal/terminal.c

```diff
@@ -1576,6 +1577,8 @@ static int __guac_terminal_send_key(guac_terminal* term, int keysym, int pressed
     /* Track modifiers */
     if (keysym == 0xFFE3)
         term->mod_ctrl = pressed;
+    else if (keysym == 0xFFE7)
```

Comment:
This is left meta/cmd only - I'd defer adding support for right-side modifiers to https://issues.apache.org/jira/browse/GUACAMOLE-1113.